dsgrid/config/dimension_config_factory.py

```diff
@@ -10,6 +11,7 @@
     DimensionType,
     AnnualTimeDimensionModel,
     RepresentativePeriodTimeDimensionModel,
+    TrivialTimeDimensionModel,
```

Comment:
Per my comment on the test data PR, I don't think you should need a new time type--AnnualTime should work (or be made to work) in this case, I think.